src/components/buttons/PopoverButton/PopoverButton.tsx

```diff
@@ -82,15 +84,20 @@ export default function PopoverButton(props: PopoverButtonProps) {
       style={{
         textTransform: 'none',
       }}
+      disabled={isDisabled}
+      aria-disabled={isDisabled}
```

Comment:
If this burtton component eventually renders `<button disabled>Text</button>` then we don't need the additional `aria-disabled`

Reply:
Woah, I accidentally typed "burtton" LOL. I'm leaving it! That's too good.

Reply:
Ah, thanks for this. I really need to up my ARIA understanding!

Reply:
Me too! I learned a bit more about it because ~I'm an thoughtful developer who cares about others~ I worked at a SaaS company that had components which were injected onto retail sites and those retailers were getting sued for not having accessible websites 😆.

A general rule of thumb is this: whenever you find yourself using elements that _emulate_ web standards, you need an aria attribute to tell user agents what this thing is _supposed_ to be. A classic example occurs when people emulate `<table>`s with `<div>`s.

```html
<div role="row">
  <div role="gridcell">Jane</div>
  <div role="gridcell">Smith</div>
  <div role="gridcell">496-619-5098</div>
  …
</div>
```

You're saying, "Hey, user agent, this isn't just any ol' div. Treat this thing like a `<table>`!"

```html
<div role="button" aria-disabled="true" tabindex="-1">Edit</div>
```

Emulating the native elements can be a dangerous game. And it's totally impossible for one or many of our funding sources to demand our HTML to meet a certain accessibility level.